src/Avalonia.Visuals/Media/GlyphTypeface.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+
+using Avalonia.Platform;
+
+namespace Avalonia.Media
+{
+    public class GlyphTypeface : IDisposable
```

Comment:
This just wrapps `IGlyphTypefaceImpl` so it should be safe to use the interface directly.